### PR TITLE
Docs updated with changes to init & serve command

### PR DESCRIPTION
**Pull request by fayeed**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,14 +22,33 @@ Get help inforamtion for available commands,
 Initialize the dependencies by coping the holochain binaries (**Required**),
 
 ```shell
+# Initializes the ad4m-host with default config
 ./ad4m init
+
+# Initalize ad4m-host with different environment
+./ad4m init --dataPath environment_name --networkBootstrapSeed ./custom_seed.json
```
Comment (kaichaosun) [inline README.md]: does the `environmen_name` mean `ad4m`?
we better give a concrete and working example here I think.

File: README.md [hunk 2]
```diff
@@ -22,14 +22,33 @@ Get help inforamtion for available commands,
...
+
+# To override the environment you can use overrideConfig flag
```
Comment (kaichaosun) [inline README.md:31]: I don't quite get it here, do you mean the overrideConfig will override all the data identified by `environment_name`?

File: README.md [hunk 3]
```diff
@@ -22,14 +22,33 @@ Get help inforamtion for available commands,
...
+./ad4m serve
+
+# will start holochain process for this environment
+./ad4m serve --dataPath environment_name
+
+# Start server with only languageLanguage and ignore other languages
+./ad4m serve --languageLanguageOnly
+
+# Pass custom bootstrap languages & perspective
+./ad4m serve --bootstrapLanguage "stringified_language_json" --bootstrapPerspective "stringified_perspective_json"
```
Comment (kaichaosun) [inline README.md]: Here we better use concrete and working example command.

File: README.md (hunk 2 above)
Comment (fayeed) [reply README.md:31]: yes

Comment (kaichaosun) [reply README.md:31]: @fayeed should we call it something like clean state?

**Review by kaichaosun — APPROVED** [review]
👍

**Pull request merged**